### PR TITLE
Disable b152292 for arm32 also

### PR DESCRIPTION
**Pull request by BruceForstall**

### CODE REVIEW

**Comment by BruceForstall** [comment]
@dotnet-bot test Ubuntu arm Cross Checked jitstress2 Build and Test
@dotnet-bot test Ubuntu16.04 arm64 Cross Checked jitstress2 Build and Test

**Comment by BruceForstall** [comment]
Issue: https://github.com/dotnet/coreclr/issues/20358

File: tests/issues.targets [hunk 1]
```diff
@@ -544,6 +544,14 @@
         </ExcludeList>
     </ItemGroup>
 
+    <!-- Unix arm64 or arm32 -->
+    <ItemGroup Condition="'$(XunitTestBinBase)' != '' and ('$(BuildArch)' == 'arm64' or '$(BuildArch)' == 'arm') and '$(TargetsWindows)' != 'true' and '$(BuildTestsAgainstPackages)' != 'true'">
+        <ExcludeList Include="$(XunitTestBinBase)/JIT/Regression/CLR-x86-JIT/v2.1/b152292/b152292/*">
+            <Issue>20358</Issue>
```
Comment (echesakovMSFT) [inline tests/issues.targets]: Do we really need another ItemGroup for arm32|arm64? 
I would just add the issue to Unix arm32 group below

Comment (BruceForstall) [reply tests/issues.targets]: Well, I was trying to avoid duplication, but I can do that instead.

**Comment by BruceForstall** [comment]
Ubuntu arm fails with the usual:
```
System.IO.FileLoadException: Could not load file or assembly 'baseservices.varargs.XUnitWrapper, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null'. An operation is not legal in the current state. (Exception from HRESULT: 0x80131509 (COR_E_INVALIDOPERATION))
```
https://ci.dot.net/job/dotnet_coreclr/job/master/job/arm_cross_checked_ubuntu_innerloop_tst_prtest/5081/consoleFull

Ubuntu arm64 fails with:
```
[/home/robox/j/workspace/dotnet_coreclr/master/arm64_cross_checked_ubuntu16.04_innerloop_tst_prtest/bin/tests/Linux.arm64.Checked/tracing/eventsource/eventsourcetrace/eventsourcetrace/eventsourcetrace.sh]: 
15:08:13 
15:08:13 
15:08:13 Assert failure(PID 11383 [0x00002c77], Thread: 11383 [0x2c77]): Consistency check failed: Crst Level violation: Can't take level 18 lock CrstPendingTypeLoadEntry because you already holding level 17 lock CrstEventPipe
15:08:13 FAILED: false
15:08:13     File: /mnt/j/workspace/dotnet_coreclr/master/arm64_cross_checked_ubuntu16.04_innerloop_prtest/src/vm/crst.cpp Line: 761
15:08:13     Image: /home/robox/j/workspace/dotnet_coreclr/master/arm64_cross_checked_ubuntu16.04_innerloop_tst_prtest/bin/tests/Linux.arm64.Checked/Tests/Core_Root/corerun
```
https://ci.dot.net/job/dotnet_coreclr/job/master/job/arm64_cross_checked_ubuntu16.04_innerloop_tst_prtest/586/consoleFull

**Pull request merged**